pkgs/development/python-modules/botorch/default.nix

```
@@ -46,7 +47,18 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [ pytestCheckHook ];
 
-  disabledTests = [ "test_all_cases_covered" ];
+  pytestFlagsArray = [
+    # tests tend to get stuck on busy hosts, increase verbosity to find out
+    # which specific tests get stuck
+    "-vvv"
+  ];
+
+  disabledTests =
+    [ "test_all_cases_covered" ]
+    ++ lib.optionals (stdenv.buildPlatform.system == "x86_64-linux") [
+      # stuck tests on hydra
+      "test_moo_predictive_entropy_search"
+    ];
 
```

I wonder if we should mark this somehow, sort of

```suggestion

  requiredSystemFeatures =  [ "benchmark" "big-parallel" ];
```

---

I agree with `big-parallel`, not so much with `benchmark`. Requiring benchmark will essentially pin it to t2a.cunat.cz, which is the only machine with that feature.